### PR TITLE
Change running user/group to root

### PR DESCRIPTION
**Pull request by blueSherii**
Issue related https://github.com/adjust/infrastructure/issues/10890

### CODE REVIEW

File: dev-db/aerospike-server-enterprise/files/aerospike.init3 [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/sbin/openrc-run
+
```
Comment (blueSherii) [inline dev-db/aerospike-server-enterprise/files/aerospike.init3:2]: user and group removed from this section .
A new init file created to preserve the old ones

Comment (einhverfr) [reply dev-db/aerospike-server-enterprise/files/aerospike.init3:2]: not sure why this is needed.  Couldn't putting relevant things in conf.d by our automation do it better?

**Review by einhverfr — CHANGES_REQUESTED** [review]
So let's sync up on this.

My thinking is the conf.d system is probably a better place for this rather than the gentoo overlay files directly and then we can create those with our automation.  Let's discuss soon

File: dev-db/aerospike-server-enterprise/aerospike-server-enterprise-4.5.0.12.ebuild [hunk 2]
```diff
@@ -65,14 +65,17 @@ src_install() {
 		doins "${FILESDIR}/${conf}"
 	done
 
+	insinto /etc/conf.d
```
Comment (monsieurp) [inline dev-db/aerospike-server-enterprise/aerospike-server-enterprise-4.5.0.12.ebuild:68]: Please use the `newconfd` function instead:
> newconfd | Install an /etc/conf.d entry using the second argument as the name

See: https://devmanual.gentoo.org/function-reference/install-functions/

**Review by monsieurp — CHANGES_REQUESTED** [review]

**Comment by blueSherii** [comment]
@einhverfr I think putting this in gentoo-overlay is better when the conf.d file is mostly static and not "server/flavour/other config" dependent

**Comment by mcharriere** [comment]
from my point of view, there is no need for this PR. The issue can be solved adding a conf.d in the infra repo. If fact since this changes the default values of user/root it will also change where root is not needed (like in aero-tracker).

**Pull request closed without merging**